### DISCUSSION TITLE
[@types/react-leaflet]

### DISCUSSION THREAD

**Discussion by michalek92** [DefinitelyTyped/DefinitelyTyped#63889] [Issues with a @types package]
The actual version contains types for an old version of react-leaflet ("@types/react-leaflet": "^2.8.2"). Is there any chance for an update
to react-leaflet 4.0?

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
react-leaflet includes its own types for versions 3.0.0 and later, so the `@types` package is no longer required. I've opened #64062 to remove its types on DefinitelyTyped

**Reply by EliteMasterEric** [reply]
Sounds good to me.
